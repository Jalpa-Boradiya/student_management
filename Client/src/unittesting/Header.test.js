import Header from "../components/header/Header";
import renderer from 'react-test-renderer';

const data = JSON.parse(localStorage.getItem('user'));

describe("Header", () => {
    it("should render without crashing", () => {
        const wrapper = renderer.create(<Header data={data} />);
        const tree = wrapper.toJSON();
        expect(tree).toMatchSnapshot();
    });
})
