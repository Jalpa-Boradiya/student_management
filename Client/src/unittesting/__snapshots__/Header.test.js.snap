// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render without crashing 1`] = `
<div
  className="header-container"
>
  <div
    className="dashboard-title"
  >
    <span>
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          {
            "color": undefined,
          }
        }
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g>
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M3 13h8V3H3v10zm0 8h8v-6H3v6zm10 0h8V11h-8v10zm0-18v6h8V3h-8z"
          />
        </g>
      </svg>
    </span>
    <h1>
      DASHBOARD
    </h1>
    <h1
      className="name-heading"
    >
      Welcome : Name
    </h1>
  </div>
  <div
    className="home-line"
  />
</div>
`;
